corehq/apps/geospatial/views.py

```diff
@@ -441,10 +441,17 @@ def get_users_with_gps(request, domain):
 @method_decorator(toggles.GEOSPATIAL.required_decorator(), name="dispatch")
 class CasesReassignmentView(BaseDomainView):
     urlname = "reassign_cases"
+    MAX_REASSIGNMENT_REQUEST_CASES = 100
 
     def post(self, request, domain, *args, **kwargs):
         request_data = json.loads(request.body)
         case_id_to_owner_id = request_data.get('case_id_to_owner_id', {})
+        if len(case_id_to_owner_id) > self.MAX_REASSIGNMENT_REQUEST_CASES:
+            return HttpResponseBadRequest(
```

Comment:
Since this API will be used only for Case Management page which supports max 100 assignment at a time, I think we should keep this limit. This also ensures we don't end up running into large no. of related cases due to large no. of request cases itself.

Reply:
That makes sense, thanks for the clarification!